app-commons/src/main/scala/org/bitcoins/commons/serializers/Picklers.scala

```diff
@@ -507,4 +507,12 @@ object Picklers {
         )
     }
   }
+
+  implicit val mnemonicCodePickler: ReadWriter[MnemonicCode] =
+    readwriter[String].bimap(
+      _.words.mkString(" "),
+      str => MnemonicCode.fromWords(str.split(' ').toVector))
+
+  implicit val extPrivateKeyPickler: ReadWriter[ExtPrivateKey] =
+    readwriter[String].bimap(_.toString, ExtPrivateKey.fromString)
```

Comment:
is this going to be a problem calling `_.toString` ? The `ExtPrivateKey extends MaskedToString`